scalameta/semantic/shared/src/main/scala/scala/meta/internal/semantic/vfs/Paths.scala

```diff
@@ -1,28 +1,26 @@
 package scala.meta.internal.semantic.vfs
 
 import scala.meta.internal.io.PathIO.fileSeparator
+import scala.meta.io.AbsolutePath
```

Comment:
Can we just enable -Ywarn-unused:import to prevent humans having to catch this? If it makes the local development cycle awkward you can either disable that flag locally, or we could modify build to enable the flag in CI runs.